test/e2e/mock-e2e-allowlist.js

```diff
@@ -75,27 +75,9 @@ const ALLOWLISTED_URLS = [
   'https://metamask.github.io/snaps/test-snaps/2.20.1',
   'https://metamask.github.io/test-dapp/metamask-fox.svg',
   'https://metamask-sdk.api.cx.metamask.io/evt',
-  'https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=ANTANI',
-  'https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=PHP,USD',
-  'https://min-api.cryptocompare.com/data/price?fsym=ETH&tsyms=USD',
-  'https://min-api.cryptocompare.com/data/price?fsym=MATIC&tsyms=USD',
-  'https://min-api.cryptocompare.com/data/price?fsym=TEST&tsyms=USD',
-  'https://min-api.cryptocompare.com/data/pricemulti?fsyms=btc%2Csol&tsyms=usd',
-  'https://min-api.cryptocompare.com/data/pricemulti?fsyms=ETH%2CMegaETH%2CTEST&tsyms=usd',
-  'https://min-api.cryptocompare.com/data/pricemulti?fsyms=ETH%2CMegaETH%2CMATIC&tsyms=usd',
-  'https://nft.api.cx.metamask.io/collections?contract=0x57f1887a8bf19b14fc0df6fd9b2acc9af147ea85&chainId=1',
-  'https://nft.api.cx.metamask.io/collections?chainId=0x1&contract=0x0000000000000000000000000000000000000002',
-  'https://nft.api.cx.metamask.io/collections?chainId=0x539&contract=0x581c3c1a2a4ebde2a0df29b5cf4c116e42945947',
-  'https://nft.api.cx.metamask.io/collections?chainId=0x539&contract=0x5fbdb2315678afecb367f032d93f642f64180aa3',
-  'https://nft.api.cx.metamask.io/users/0x5cfe73b6021e818b776b421b1c4db2474086a7e1/tokens?chainIds=1&limit=50&includeTopBid=true&continuation=',
```

Comment:
the nft mocks were not correctly applied (and one was missing). These has been updated, so now we don't rely on these live reqs anymore